### PR TITLE
Bug fixs + Improvements

### PR DESCRIPTION
**Pull request by ghost**
* Halved the max_packet limit. (Improves the chances the server will be okay if it gets flooded)
* Wires now die when they hit max_packet, and a packet blows up (shows an effect)
* Children of signal no longer continue processing if the packet was deleted

* Labs no longer try to load labs without a save file
* Previously existing labs will no longer load twice when LoadAllLabs is pressed

* Start area is now visible in the editor

### CODE REVIEW

**Review by Nadrew — CHANGES_REQUESTED** [review]
This should be an fexists() call, not length(file()).

File: code/lab_saving/main.dm [hunk 1]
```diff
@@ -77,6 +77,11 @@ area
 				F["lab"] << src
 
 			Load(mob/caller)
+				var/save_name = "saves/labs/new/[ckey(src.name)].lab"
+				if(length(file(save_name))==0)
```
Comment (Nadrew) [inline code/lab_saving/main.dm]: Right here, fexists().

**Review by Nadrew — CHANGES_REQUESTED** [review]

File: code/signal/signal.dm [hunk 2]
```diff
@@ -28,6 +28,11 @@ obj/signal
 			max_signal = 150
 	process_signal(obj/signal/structure/S,atom/source)
 		if(signal_hit>=max_signal)
+			if(istype(source,/obj/signal/wire))
+				var /area/boom_loc = source.loc
```
Comment (Nadrew) [inline code/signal/signal.dm]: Would be more efficient to use an image here instead of an object.

**Review by Nadrew — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Okay, changed.

**Review by Nadrew — APPROVED** [review]

**Pull request merged**